.env.example

```diff
@@ -0,0 +1,3 @@
+AWS_ACCESS_KEY_ID=
```

Comment:
good work! this will give other devs hints as to what credentials they'll require when they set up their machine to do active development on this application